jekyll/_cci2/create-an-orb.adoc

```diff
@@ -30,37 +30,37 @@ video::5ta4RUwqOBI[youtube]
 [#prerequisites]
 == Prerequisites
 
-* A GitHub account and connected CircleCI account. See the link:/docs/first-steps/[Sign up and try] page for steps to get set up.
-* link:/docs/local-cli/#installation[Set up the CircleCI CLI] on your local machine, with a link:https://app.circleci.com/settings/user/tokens[personal access token]. Ensure you are using the latest version of the CircleCI CLI. You must be using version `v0.1.17087` or later.
+* A GitHub account connected to your CircleCI account. See the link:/docs/first-steps/[sign up and try] page for steps to get set up.
+* link:/docs/local-cli/#installation[set up the CircleCI CLI] on your local machine with a link:https://app.circleci.com/settings/user/tokens[personal access token]. Ensure you are using the latest version of the CircleCI CLI. You must be using version `v0.1.17087` or later.
```

Comment:
Is there a reason you changed the beginning letter of the sentence to lowercase? It makes more sense for it to be uppercase in the list format

Reply:
No reason,  just 🙃, thanks!